flask_restplus_patched/parameters.py

```diff
@@ -58,6 +59,9 @@ class PatchJSONParameters(Parameters):
     op = base_fields.String(required=True) # pylint: disable=invalid-name
 
     PATH_CHOICES = None
+
+    NO_VALUE_OPERATIONS = ('remove',)
```

Comment:
1. There is `OP_REMOVE` const
2. Please, put this declaration right before `value` field. I grouped these things together.